requirements/edx/github.txt

```diff
@@ -90,7 +90,7 @@ git+https://github.com/edx/xblock-utils.git@v1.0.3#egg=xblock-utils==1.0.3
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@138e6fa0bf3a2013e904a085b9fed77dab7f3f21#egg=xblock-google-drive
 -e git+https://github.com/edx/edx-reverification-block.git@0.0.5#egg=edx-reverification-block==0.0.5
 git+https://github.com/edx/edx-user-state-client.git@1.0.1#egg=edx-user-state-client==1.0.1
-git+https://github.com/edx/xblock-lti-consumer.git@v1.1.1#egg=xblock-lti-consumer==1.1.1
+git+https://github.com/edx/xblock-lti-consumer.git@v1.1.1#egg=lti_consumer-xblock==1.1.1
```

Comment:
My understanding is that the egg name here is purely so that pip can pre-check whether it needs to get the distribution.  Once the repo is downloaded, all information (name, version) comes from the setup.py.